docs/project_digital_bone_box/about.md

```diff
@@ -0,0 +1,82 @@
+---
+id: about
+title: Digital Bone Box
+custom_edit_url: null
+---
+
+<!-- A header image is optional; if used should be no greater than 200x600 -->
+<!--![Header Alt Text](header.png) -->
+
+## Overview
+
+The Digital Bone Box Web App project aims to convert an existing PowerPoint-based educational tool into an interactive, mobile-friendly web application. This app will provide medical students with a comprehensive, easy-to-navigate resource for studying skeletal anatomy. 
+
+The app will categorize the bones into modules such as the bony pelvis, lower limb, upper limb, thorax, skull, and vertebrae. Each module will allow students to explore different bone structures, view detailed information, and interact with labels that identify various parts of the bones. 
+
+### Information
+
+- **Source Code:** -
+- **Client** Dr. Brian Elliot - Program Director - Clinical Health Sciences
+- **Current Tech Lead:** Devi Priya Kolla
```

Comment:
Make sure you get at least your GitHub profile linked here, and possibly your LinkedIn profile. there are examples for the markdown code to do this on the existing pages.

Reply:
Linked my GitHub and LinkedIn profiles.